pyopencl/cl/pyopencl-ranluxcl.cl

```diff
@@ -320,7 +320,7 @@ double4 ranluxcl64norm(ranluxcl_state_t *);
 #endif //RANLUXCL_LUX == 0
 
 //Check that nskip is a permissible value
-#if RANLUXCL_NSKIP % 4 != 0 
```

Comment:
RANLUX and Random123 are "vendored", so I'd prefer to keep them unmodified, typos and all.

Reply:
Ah, good to know! I've reverted those in the latest version.